atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -236,10 +238,32 @@ def _make_labels(self):
 
         return labels
 
+    def get_fs_task_id(self):
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
```

Comment:
If the add_filesystem plugin ran, do we want to fail if the task_id cannot be determined?
Meaning, should we silence the `KeyError` and `ValueError` exceptions here?